docs/Adding-Software-to-Homebrew.md

```diff
@@ -175,13 +175,13 @@ This will open `EDITOR` with a template for your new cask, to be stored in the f
 
```

Comment:
I think it's probably worth removing this template instead of trying to keep it up-top-date but not consistent with `brew create`.

Reply:
Yeah let's remove the template.